### PR TITLE
ocl-icd: 2.2.10 -> 2.3.1

### PR DESCRIPTION
**Pull request by trofi**

### CODE REVIEW

File: pkgs/development/libraries/ocl-icd/default.nix [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+, ruby
+, opencl-headers
+, addOpenGLRunpath
+, autoreconfHook
+}:
 
 stdenv.mkDerivation rec {
   pname = "ocl-icd";
-  version = "2.2.10";
+  version = "2.2.13";
```
Comment (figsoda) [inline pkgs/development/libraries/ocl-icd/default.nix:12]: 2.3.1 is out, any reason to not use the latest version?

File: pkgs/development/libraries/ocl-icd/default.nix [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
 
   src = fetchurl {
-    url = "https://forge.imag.fr/frs/download.php/810/${pname}-${version}.tar.gz";
-    sha256 = "0f14gpa13sdm0kzqv5yycp4pschbmi6n5fj7wl4ilspzsrqcgqr2";
+    url = "https://github.com/OCL-dev/ocl-icd/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "14ld3xmkgf1755xn937nxzd997l2y26sxr4igdiiazrjx3rmjpgq";
   };
```
Comment (figsoda) [inline pkgs/development/libraries/ocl-icd/default.nix:17]: use `fetchFromGitHub` instead

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/ocl-icd/default.nix (hunk 1 above)
Comment (trofi) [reply pkgs/development/libraries/ocl-icd/default.nix]: I wanted to cheat a bit and rely on the bot for automatic update once we point it an up to date repo :) Split commit in two: switch homepage without release increment and updated release separately (in unlikely case of the needed revert).

**Comment by trofi** [comment]
Should be ready for review.

**Review by figsoda — APPROVED** [review]
lgtm, i would still like another reviewer though

~~my nixpkgs-review is still running, wine and blender are taking a long time~~

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 139969` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.pymc3</li>
    <li>python39Packages.pymc3</li>
    <li>scallion</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>forge</li>
  </ul>
</details>
<details>
  <summary>60 packages built:</summary>
  <ul>
    <li>arrayfire</li>
    <li>beignet</li>
    <li>blender</li>
    <li>bumblebee</li>
    <li>cgminer</li>
    <li>cl</li>
    <li>clblas</li>
    <li>clfft</li>
    <li>clinfo</li>
    <li>clmagma</li>
    <li>clpeak</li>
    <li>computecpp</li>
    <li>computecpp-unwrapped</li>
    <li>coretoppings</li>
    <li>darktable</li>
    <li>driversi686Linux.beignet</li>
    <li>ethminer (ethminer-cuda)</li>
    <li>ethminer-free</li>
    <li>fahclient</li>
    <li>geekbench</li>
    <li>hashcat</li>
    <li>katago</li>
    <li>leela-zero</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>luxcorerender</li>
    <li>mandelbulber</li>
    <li>ocl-icd</li>
    <li>opencl-info</li>
    <li>opensubdiv</li>
    <li>pipelight</li>
    <li>playonlinux</li>
    <li>pyrit</li>
    <li>python38Packages.Theano (python38Packages.TheanoWithoutCuda)</li>
    <li>python38Packages.TheanoWithCuda</li>
    <li>python38Packages.libgpuarray</li>
    <li>python38Packages.loo-py</li>
    <li>python38Packages.pyopencl</li>
    <li>python38Packages.reikna</li>
    <li>python38Packages.sasmodels</li>
    <li>python39Packages.Theano (python39Packages.TheanoWithoutCuda)</li>
    <li>python39Packages.TheanoWithCuda</li>
    <li>python39Packages.libgpuarray</li>
    <li>python39Packages.loo-py</li>
    <li>python39Packages.pyopencl</li>
    <li>python39Packages.reikna</li>
    <li>python39Packages.sasmodels</li>
    <li>sdrangel</li>
    <li>sycl-info</li>
    <li>virtualgl</li>
    <li>virtualglLib</li>
    <li>waifu2x-converter-cpp</li>
    <li>wf-recorder</li>
    <li>wifite2</li>
    <li>wine (winePackages.full)</li>
    <li>wine-staging</li>
    <li>wineWowPackages.full</li>
    <li>winetricks</li>
    <li>wings</li>
    <li>xmr-stak</li>
  </ul>
</details>

**Comment by figsoda** [comment]
forge already fails to build on master

**Pull request merged**